packages/hash/frontend/src/components/RemoteBlock/RemoteBlock.tsx

```diff
@@ -134,7 +144,9 @@ export const RemoteBlock: FunctionComponent<RemoteBlockProps> = ({
           properties={propsToInject}
           sourceUrl={blockMetadata.source}
         />
-      ) : null}
+      ) : (
+        <BlockLoadingIndicator />
+      )}
```

Comment:
I thought instead of rendering null for a moment, we can keep rendering the loading state, let me know if there is sth I'm wrong about this @nathggns